cmd/zarb/start.go

```diff
@@ -38,6 +40,10 @@ func Start() func(c *cli.Cmd) {
 			Name: "a auth",
 			Desc: "Passphrase of the key file",
 		})
+		pprofOpt := c.String(cli.StringOpt{
+			Name: "d pprof",
+			Desc: "debug pprof server address Address(not recommended to expose to internet)",
```

Comment:
I suggest this:
```
Name: "pprof",			
Desc: "debug pprof server address (not recommended to expose to internet)",
```

- Remove one letter option
- Address address -> address

Reply:
applyed